src/app/qgsattributetabledialog.cpp

```diff
@@ -1042,6 +1043,21 @@ void QgsAttributeTableDialog::setFilterExpression( const QString &filterString,
 }
 
 
+void QgsAttributeTableDialog::deleteFeature( const QgsFeatureId &fid )
```

Comment:
The parameter should be passed by value `QgsFeatureId fid`, this is actually a simple `long long` which is faster to pass by value than by ref.